src/transformers/models/xlm_roberta_xl/configuration_xlm_roberta_xl.py

```diff
@@ -0,0 +1,154 @@
+# coding=utf-8
+# Copyright 2018 The Google AI Language Team Authors and The HuggingFace Inc. team.
+# Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" XLM-RoXLM_ROBERTA_XLa configuration"""
+
+from collections import OrderedDict
+from typing import Mapping
+
+from ...configuration_utils import PretrainedConfig
+from ...onnx import OnnxConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+XLM_ROXLM_ROBERTA_XLA_XL_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    # to fill
+}
+
+
+class XLMRobertaXLConfig(PretrainedConfig):
```

Comment:
The default values need to be changed to https://huggingface.co/Soonhwan-Kwon/xlm-roberta-xlarge/blob/main/config.json

Reply:
changed default values as config, thank you for the heads up!